pkg/build/builder/util.go

```diff
@@ -4,6 +4,11 @@ import (
 	buildapi "github.com/openshift/origin/pkg/build/api"
 )
 
+const (
+	// DefaultDockerLabelNamespace is the key of a Build label, whose values are build metadata.
+	DefaultDockerLabelNamespace = "io.openshift."
```

Comment:
there's an "io.openshift.s2i." in s2i, that's probably what @mfojtik was thinking of.